x-pack/test/functional/apps/api_keys/home_page.ts

```diff
@@ -13,11 +13,26 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
   const log = getService('log');
   const security = getService('security');
   const testSubjects = getService('testSubjects');
+  const es = getService('es');
   const find = getService('find');
   const browser = getService('browser');
 
+  async function clearAllApiKeys() {
+    const existingKeys = await es.security.queryApiKeys();
+    if (existingKeys.count > 0) {
+      Promise.all(
+        await existingKeys.api_keys.map(async (key) => {
+          await es.security.invalidateApiKey({ ids: [key.id] });
```

Comment:
At the same time, there is no need to use async/await for internal operation since `invalidateApiKey` is already defined as async function.
Basically the proper solution afaik is:

```
  async function clearAllApiKeys() {
    const existingKeys = await es.security.queryApiKeys();
    if (existingKeys.count > 0) {
      await Promise.all(
        existingKeys.api_keys.map((key) => {
           es.security.invalidateApiKey({ ids: [key.id] });
        })
      );
    } else {
      log.debug('No API keys to delete.');
    }
  }
```

Reply:
Fixed.